metis-indexing/metis-indexing-service/src/main/java/eu/europeana/indexing/model/IndexingMongoServer.java

```diff
@@ -0,0 +1,22 @@
+package eu.europeana.indexing.model;

```

Comment:
Ok so I see that in the FullBeanDao the connection to the database is initialized.
The connection to the database should be initialized in a different class, maybe IndexingMongoServer. (Similar way to MorphiaDatastoreProvider, search on the project is in the metis-core)

And the FullBeanDao which is a DAO should be provided with a connection and that connection should be used for any DAO operation(DatasetDao).

Reply:
I don't think this class is used properly at all! I think it should just be removed as it is not necessary. See my comments in PublishingService (the only place where it is being used).